api/proxy/.golangci.yml

```diff
@@ -195,7 +195,7 @@ linters:
           - unparam
 
       # Allow certain patterns to be ignored by lll (long lines)
-      - source: '".{120,}"' # Ignores double-quoted strings longer than 120 chars
+      - source: '".{100,}"' # Ignores double-quoted strings longer than 120 chars
```

Comment:
Not in favor of this change

Reply:
I know... but otherwise there was a bug which the celo engineer was running into and he didn't find another way to solve it. See https://eigenlabs.slack.com/archives/C07614NNXDH/p1751913947976629?thread_ts=1750965737.272329&cid=C07614NNXDH

Reply:
Fixed comment and added some "explanation" in [2c50038](https://github.com/Layr-Labs/eigenda/pull/1776/commits/2c500384b1392f2bcdce0d4a687b8256288598ca)
Tbf I don't understand the bug so not super sure... but its not super important to me we can keep tweaking this in another PR I think?